frontend/src/i18n/locales/en/chat.json

```
@@ -97,6 +97,28 @@
     "collapse_content": "Collapse",
     "lines": "lines"
   },
+  "thinking": {
```

Biome (2.3.13)

[error] The key thinking was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)